### DISCUSSION TITLE
PhysX binaries for Windows x64

### DISCUSSION THREAD

**Discussion by thevbw** [NVIDIA-Omniverse/PhysX#233] [Q&A]
I want to build a Windows version of my application, but have failed to cross-compile the PhysX library as the heavy build system conflates Windows with MSVC and I am not familiar enough with CMake to adapt the scripts. Are official pre-built binaries available? It builds correctly for Linux.

**Comment by preist-nvidia** [comment]
Hi @thevbw - this might be helpful: https://omni-101.github.io/omniverse/2023/07/07/omniverse-physx-vcpkg-port/
